<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptUnpacker</class>
 <widget class="QDialog" name="ScriptUnpacker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script Decompiler</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page_v3v4">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <widget class="QCheckBox" name="sepFolders">
         <property name="text">
          <string>Seperate Extraction Folders (always active if using custom aliases)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="useHex">
         <property name="text">
          <string>Use Hexadecimal Values (will not recompile, but helps for readability)</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="useCustomAliases">
         <property name="text">
          <string>Use Extra Aliases (will not recompile, but helps for readability)</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="selDataFolder">
         <property name="text">
          <string>Select Data Folder</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="selExtractFolder">
         <property name="text">
          <string>Select Extraction Folder</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="useOldOps">
         <property name="text">
          <string>Use Old Opcode List (v4 only)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_v1">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>71</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="loadTRBytecode">
         <property name="text">
          <string>Select TaxReceipt Bytecode File</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="saveTRScript">
         <property name="text">
          <string>Save TaxReceipt Script</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>71</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="selEngineType">
     <item>
      <property name="text">
       <string>RSDKv3 (Sonic CD)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RSDKv4 (Sonic 1/Sonic 2)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RSDKv1 (Retro-Sonic)</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
